src/components/label/ohri-label.component.tsx

```diff
@@ -1,17 +1,24 @@
 import React from 'react';
 import { DefinitionTooltip } from '@carbon/react';
 import styles from './ohri-label.scss';
+import { useTranslation } from 'react-i18next';
+import i18next from '../translator/i18n';
 
 type LabelProps = {
   value: string;
   tooltipText?: string;
 };
 
 export const OHRILabel: React.FC<LabelProps> = ({ value, tooltipText }) => {
+  const { t, i18n } = useTranslation('ohri', { i18n: i18next });
+  const valueTranslated = t('value', value);
+
+  //TODO: Missing button for component change language instead of default language
```

Comment:
My understanding is that the form-engine should not introduce any button but use what is defined by the app, I believe we have a language switcher button somewhere in O3.

Reply:
And even if we decided to add a button it would not be at the label level but rather at the form level